### PR TITLE
Update Cypress test to cover "Intelligent scissors blocking feature".

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Update Cypress test to check "Intelligent scissors blocking feature". PR #3510 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by bsekachev** [comment]
@klakhov Please, review

File: tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js [hunk 1]
```diff
@@ -88,6 +88,12 @@ context('OpenCV. Intelligent cissors. Histogram Equalization.', () => {
             cy.get('.cvat_canvas_interact_intermediate_shape').should('have.attr', 'fill-opacity', 0);
             cy.get('.cvat-appearance-selected-opacity-slider').click('right');
             cy.get('.cvat_canvas_interact_intermediate_shape').should('have.attr', 'fill-opacity', 1);
+            // Intelligent scissors blocking feature
+            cy.get('.cvat_canvas_threshold').should('exist');
+            cy.contains('span', 'Block').click();
+            cy.get('.cvat_canvas_threshold').should('not.exist');
+            cy.contains('span', 'Block').click();
+            cy.get('.cvat_canvas_threshold').should('exist');
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js:96]: Please, check that the block actually work (it creates exactly line user draws without adding any new points)

Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks2/case_101_opencv_basic_actions.js]: Reworked. I allocated it to a separate test. Added checks.

**Review by klakhov — APPROVED** [review]
I agree that we need to test amount of points. Seems good.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**